tests/unit/tf/horovod/test_horovod.py

```diff
@@ -0,0 +1,79 @@
+import importlib
+import os
+
+import tensorflow as tf
+
+import merlin.models.tf as mm
+from merlin.datasets.advertising.criteo.dataset import default_criteo_transform
+from merlin.io.dataset import Dataset
+from merlin.models.tf.distributed.backend import hvd
+from merlin.models.utils.example_utils import workflow_fit_transform
+from merlin.schema.tags import Tags
+
+
+def test_import():
```

Comment:
Why is this test written this way? Doesnt the init file pytest.importorskip cover this? In what case does this assert fail?

Reply:
It was a silly test carried over from template code. Removed.